spec/feature_flagger/model_spec.rb

```diff
@@ -19,30 +19,66 @@ def id; 14 end
     end
 
     describe '#release!' do
-      it 'calls Control#release! with appropriated methods' do
-        expect(control).to receive(:release!).with(key, subject.id, 'feature_flagger_dummy_class')
-        subject.release!(key)
+      context 'with key as multiple arguments' do
+        it 'calls Control#release! with appropriated methods' do
+          expect(control).to receive(:release!).with(key, subject.id, 'feature_flagger_dummy_class')
+          subject.release!(*key)
+        end
+      end
+
+      context 'with key argument as an array' do
+        it 'calls Control#release! with appropriated methods' do
+          expect(control).to receive(:release!).with(key, subject.id, 'feature_flagger_dummy_class')
+          subject.release!(key)
+        end
       end
     end
 
     describe '#rollout?' do
-      it 'calls Control#rollout? with appropriated methods' do
-        expect(control).to receive(:rollout?).with(key, subject.id, 'feature_flagger_dummy_class')
-        subject.rollout?(key)
+      context 'with key as multiple arguments' do
+        it 'calls Control#rollout? with appropriated methods' do
+          expect(control).to receive(:rollout?).with(key, subject.id, 'feature_flagger_dummy_class')
+          subject.rollout?(*key)
+        end
+      end
+
+      context 'with key argument as an array' do
```

Comment:
IMHO adding this extra context is like to test Ruby behavior over _variadic methods_, same for the '*'. We assume that this ruby feature is working properly. right?

Does that make sense?